tests/integration/jsonschema/jsonschema.go

```diff
@@ -0,0 +1,44 @@
+package jsonschema
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/xeipuuv/gojsonschema"
+)
+
+// Validate validates the input argument against JSON schema. If the
+// input is not valid the error is returned. The first argument is the file name
+// of the JSON schema. It is used to build file URI required to load the JSON schema.
+// The second argument is the input string that is validated.
+func Validate(fileName string, input string) error {
```

Comment:
It would be nice to have a test for this public function

Reply:
Same as above